.php-cs-fixer.dist.php

```diff
@@ -12,6 +12,7 @@
 $config
     ->getFinder()
     ->ignoreDotFiles(false)
+    ->notPath('src/Contract/Operation/Allable.php')
```

Comment:
the issue with php-cs-fixer is that the rule [no_superfluous_phpdoc_tags](https://cs.symfony.com/doc/rules/phpdoc/no_superfluous_phpdoc_tags.html) removes the `@param bool $normalize` phpdoc annotation, which is needed for the conditional return type.

So here we have a choice between:
- deactivating the rule for the whole project
- ignoring that one file

Note that php-cs-fixer does not currently have a mechanism to ignore a rule for a specific file or line, like phpcs has. There are a few issues about this:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3871
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3846

Reply:
Oh. That's very unfortunate.

We could either remove that specific rule, or remove the file. What's the best choice?

Reply:
I think I would prefer to just remove the file for now, it means that we're still getting the benefit of the rule in most of the project. And even with php-cs-fixer ignoring the file we still have phpcs which will enforce style things on it

Reply:
I also think this is the best option.

Reply:
It's not mandatory because we are using Git, but how about adding a comment inside the PHPCS configuration file pointing to this PR as a reference for later?